docs/reference/commandline/run.md

```diff
@@ -198,8 +198,13 @@ The `-w` lets the command being executed inside directory given, here
     $ docker run -it --storage-opt size=120G fedora /bin/bash
 
 This (size) will allow to set the container rootfs size to 120G at creation time.
-User cannot pass a size less than the Default BaseFS Size. This option is only
```

Comment:
See below: For the `overlay2` storage driver, the size option is only available if the backing fs is `xfs`. You cannot use the size option for xfs + overlay2